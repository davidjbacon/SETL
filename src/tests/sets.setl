a := {['b',2]};
a{'b'} := {1,2,3};
print(#a);
print(a);
a{'b'} := {};
print(#a);
print(a);
