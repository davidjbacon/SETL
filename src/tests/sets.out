3
{[b 1] [b 2] [b 3]}
0
{}
